output_template/python/usb_camera_demo.py

```diff
@@ -261,10 +261,14 @@ def run(model, config_file):
 @click.command(context_settings=dict(help_option_names=['-h', '--help']))
 @click.option(
     "-m",
+    "-l",
     "--model",
     type=click.Path(exists=True),
-    help=u"Inference Model filename",
-    default="../models/lib/lib_fpga.so",
+    help=u"""
+        Inference Model filename
+        (-l is deprecated please use -m instead)
+    """,
+    default="../models/minimal_graph_with_shape.pb",
```

Comment:
I think default value is better to use FPGA lib. why do you change default value?

Reply:
@ruimashita Oh. I see. There was no reason why I changed it.  This was a mistake. Thanks!

Reply:
This was not changed :sob::sob::sob:

Reply:
@Joeper214 Could you fix this ??

Reply:
@ruimashita Sure. I'll do that.